frontend/src/components/svgs/index.js

```diff
@@ -0,0 +1,118 @@
+import * as React from "react";
+
+const ButtonArrow = () => (
+  <svg
+    className="w-8"
+    viewBox="0 0 21 13"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <g>
+      <path
+        className="fill-Accent-500"
+        d="M20.5303 7.03033C20.8232 6.73744 20.8232 6.26256 20.5303 5.96967L15.7574 1.1967C15.4645 0.903806 14.9896 0.903806 14.6967 1.1967C14.4038 1.48959 14.4038 1.96447 14.6967 2.25736L18.9393 6.5L14.6967 10.7426C14.4038 11.0355 14.4038 11.5104 14.6967 11.8033C14.9896 12.0962 15.4645 12.0962 15.7574 11.8033L20.5303 7.03033ZM0 7.25H20V5.75H0V7.25Z"
+      />
+    </g>
+  </svg>
+);
+
+const RightAngleTriangle = ({ children, side }) => (
+  <svg
+    className="w-20"
+    viewBox="0 0 80 200"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    {
+      {
+        left: (
+          <path
+            className="fill-Primary-500"
+            d="M-5.96046e-07 0V200H80L-5.96046e-07 0Z"
+          />
+        ),
+        right: <path className="fill-Primary-500" d="M80 0V200H0L80 0Z" />,
+      }[side]
+    }
+  </svg>
+);
+
+const Arrows = ({ width, viewBox, direction, size }) => (
```

Comment:
Is there a reason not to just make this one giant SVG that does the same thing as the `go-pattern` previously, instead of having to build it from the various north, east, south, west directions? Are we ever going to use these arrows anywhere else?

Reply:
Figma only gave me  separate SVGs  for the ones except for the go pattern one.  I just use the SVG code from the file to produce these.  if they were one SVG it would be easier, but they weren't.

Reply:
Ah I see